Update meta descriptions for 25 most popular Learn pages (#1370)
astronomer/docs: learn/what-is-a-hook.md

---
title: "Airflow hooks"
sidebar_label: "Hooks"
id: what-is-a-hook
---

<head>

>   <meta name="description" content="Learn about hooks and general guidelines on using them in Apache Airflow. See an example of implementing two different hooks in a DAG." />

Reviewer: ```suggestion
  <meta name="description" content="Learn about hooks and how they should be used in Apache Airflow. See an example of implementing two different hooks in a DAG." />
```

>   <meta name="og:description" content="Learn about hooks and general guidelines on using them in Apache Airflow. See an example of implementing two different hooks in a DAG." />

Reviewer: ```suggestion
  <meta name="og:description" content="Learn about hooks and how they should be used in Apache Airflow. See an example of implementing two different hooks in a DAG." />
```
